test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -8,6 +9,164 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
 {
     public class LibmanTestsUtility
     {
+        private static HashSet<string> GetTopLevelDirectoriesAndFiles(string cwd, bool caseInsensitive)
+        {
+            HashSet<string> topLevelItems = caseInsensitive ? new HashSet<string>(StringComparer.OrdinalIgnoreCase) : new HashSet<string>();
+            var dir = new DirectoryInfo(cwd);
+
+            if(dir.Exists)
+            {
+                foreach(FileSystemInfo item in dir.EnumerateDirectories())
+                {
+                    topLevelItems.Add(item.Name);
+                }
+                foreach(FileSystemInfo item in dir.EnumerateFiles())
+                {
+                    topLevelItems.Add(item.Name);
+                }
+            }
+
+            return topLevelItems;
+        }
+
+        public static void WaitForRestoredFiles(string cwd, IEnumerable<string> expectedFilesAndFolders, bool caseInsensitive, int timeout = 10000)
```

Comment:
>cwd [](start = 55, length = 3)

don't use abbreviated variable names

Reply:
will change

---
In reply to: [194548392](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194548392) [](ancestors = 194548392)